Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -1,186 +1,8 @@
-﻿@{
-    Layout = "~/_Layout.cshtml";
-    Page.Title = "Diagnostic Console";
+﻿@if (Kudu.Core.Helpers.OSDetecter.IsOnWindows())
+{
+    @RenderPage("~/DebugConsole/WindowsConsole.cshtml");

```

Comment:
`@RenderPage("WindowsConsole.cshtml")` would be cleaner to avoid baking assumptions on the folder name.